### ISSUE TITLE
Update breaks all sorting

### ISSUE THREAD

**Issue by cars11** [AdiAddons/AdiBags#1007]
### Addon Version

1.10.25

### World of Warcraft Version

Classic

### Describe the bug

Sorting stopped working overnight with last night's client update and 99% of items are all under "Equipment"

### Steps to reproduce

Login

### Verification

- [X] I have disabled all other addons and made sure this bug is triggered only with AdiBags enabled

**Comment by zazomazzo** [comment]
Just wanted to confirm. Everything went fine until todays client update (WoW Classic SoD Update 1.15.1 to 1.15.2)
Now at least all trade goods/consumables/reagents are listed under "Equipment".

For example:
![image](https://github.com/AdiAddons/AdiBags/assets/912655/07ff29cc-5039-4e12-b627-48a82276bb63)

**Comment by rodalpho** [comment]
Yep, broken in modern WoW also. Unclear if it'll be fixed at this point honestly.

**Comment by cars11** [comment]
> Yep, broken in modern WoW also. Unclear if it'll be fixed at this point honestly.

"but no new feature requests or bugs will be fixed unless they are game breaking, **i.e. a new patch breaks the addon**"

**Comment by rodalpho** [comment]
Sure, but it's been a couple of weeks and the addon is still broken.

**Comment by Cidan** [comment]
This is a bug in Blizzard's API. I'm going to take a look at some things later today and see if I can come up with a work around, though it might lead to a loss of functionality for AdiBags.

Sit tight, folks.

**Comment by rodalpho** [comment]
Awesome! Thanks for all your work Cidan, I know you have little interest in maintaining Adibags when you're putting so much (also appreciated) effort into its replacement, but many of us are still on it until BetterBags comes close to parity.

**Comment by leggolem** [comment]
It's not ideal, but a workaround for now is to uncheck Equipment in the filters.
![image](https://github.com/AdiAddons/AdiBags/assets/166519390/273cf39a-e78e-4104-9492-744f440b643e)

**Comment by cars11** [comment]
> It's not ideal, but a workaround for now is to uncheck Equipment in the filters

that is a huge help. thanks
